### DISCUSSION TITLE
Add a profile for teamviewer

### DISCUSSION THREAD

**Discussion by Micha-Btz** [netblue30/firejail#4432] [Q&A] [answered]
Hey,

I would like to create a profile for teamviewer. I looked around and found some hints. 
https://github.com/netblue30/firejail/issues/825#issuecomment-251221645
https://github.com/netblue30/firejail/issues/1824

Has something changed in firejail to that sandboxing for teamviewer is now possible?

```
firejail teamviewer 
Reading profile /home/mdomann/.config/firejail/teamviewer.profile
Reading profile /etc/firejail/disable-common.inc
Reading profile /etc/firejail/disable-passwdmgr.inc
Reading profile /etc/firejail/disable-programs.inc
Warning: networking feature is disabled in Firejail configuration file
Parent pid 12789, child pid 12790
Child process initialized in 45.95 ms

Init...
CheckCPU: SSE2 support: yes
Checking setup...
Launching TeamViewer ...
Starting network process (no daemon)
Network process already started (or error)
Launching TeamViewer GUI ...

Parent is shutting down, bye...
```
thanks Micha

**Comment by rusty-snake** [comment, marked as answer]
> Has something changed in firejail to that sandboxing for teamviewer is now possible?

No there's still nothing like a `shared-pidns` command.

**Comment by Micha-Btz** [comment]
ok thanks.
